eisbuk-admin/src/components/atoms/AttendanceCard/__tests__/AttendanceCard.test.tsx

```diff
@@ -2,43 +2,30 @@ import React from "react";
 import { render, screen } from "@testing-library/react";
 import AttendanceCard from "../AttendanceCard";
 
-import * as attendanceOperations from "@/store/actions/attendanceOperations";
-import { Category } from "eisbuk-shared";
-import { dummySlot } from "../__testData__/dummyData";
-
-const mockMarkAttImplementation = (customerId: string, attended: boolean) => ({
-  customerId,
-  attended,
-});
-jest
-  .spyOn(attendanceOperations, "markAttendance")
-  .mockImplementation(mockMarkAttImplementation);
+import { Category, SlotType } from "eisbuk-shared";
+import { customersSlot } from "../__testData__/dummyData";
+import i18n from "i18next";
 
 const mockDispatch = jest.fn();
 jest.mock("react-redux", () => ({
   useDispatch: () => mockDispatch,
 }));
+
+const spyT = jest.spyOn(i18n, "t");
+beforeEach(() => spyT.mockClear());
 describe("AttendanceCard", () => {
   describe("Smoke test", () => {
     beforeEach(() => {
-      render(<AttendanceCard {...dummySlot} />);
+      render(<AttendanceCard {...customersSlot} />);
     });
     test("should render props", () => {
       screen.getByText("13:00 - 15:00");
       screen.getByText("Saul");
       screen.getByText("Walter");
-      screen.getByText(Category.Competitive);
-    });
-  });
-  describe("Test marking attendance functionality", () => {
-    beforeEach(() => {
-      render(<AttendanceCard {...dummySlot} />);
-    });
-    test("should dispatch markAttendance with correct args", () => {
-      screen.getByText("👎").click();
-      expect(mockDispatch).toHaveBeenCalledWith(
-        mockMarkAttImplementation("heisenberg", true)
-      );
+      // TODO: move back to getByText when i18next is implemented
```

Comment:
Please leave the TODO like this: `/** @TODO */`. It's cool if you hava an addon painting the TODOs automatically, but not everyone looking at the code will have it, and this way is default by VSCode. Plus, by using this format, you can write any string with `@` sign, for instance `/** @TEMP @Ivan :  */`, etc. and GitHub (which has really simple code highlighting recognizes the `/** @Something */` and colors it differently.

Reply:
TIL - Good to know! I guess this is to support jsdoc syntax.